source/navitiacommon/navitiacommon/models.py

```diff
@@ -507,6 +507,15 @@ def __init__(self, name=None, street=None, address=None,
         self.poi = poi
         self.admin = admin
 
+    def main_dir(self, root_path):
+        return '{}/{}'.format(root_path, self.name)
+
+    def source_dir(self, root_path):
+        return '{}/{}/{}'.format(root_path, self.name, "source")
```

Comment:
minor: you can use as previously os.path.join(root_path, self.name, "source") it will handle the '/' a bit better